ERROR - 2022-12-03 02:34:50 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('localhost', 'root', '', 'financialdb', 3306, '', 0)
#1 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\Model.php(250): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\BaseModel.php(623): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\FinancialManagementSystem\app\Controllers\SigninController.php(21): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\SigninController->loginAuth()
#8 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SigninController))
#9 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\FinancialManagementSystem\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(46): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\Model.php(250): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\BaseModel.php(623): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\FinancialManagementSystem\app\Controllers\SigninController.php(21): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\SigninController->loginAuth()
#7 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SigninController))
#8 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\FinancialManagementSystem\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(46): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2022-12-03 02:34:50 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(250): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(623): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\SigninController.php(21): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\SigninController->loginAuth()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SigninController))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
ERROR - 2022-12-03 07:18:14 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('localhost', 'root', '', 'financialdb', 3306, '', 0)
#1 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\Model.php(250): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\BaseModel.php(623): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\FinancialManagementSystem\app\Controllers\SigninController.php(21): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\SigninController->loginAuth()
#8 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SigninController))
#9 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\FinancialManagementSystem\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(46): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\Model.php(250): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\BaseModel.php(623): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\FinancialManagementSystem\app\Controllers\SigninController.php(21): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\SigninController->loginAuth()
#7 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SigninController))
#8 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\FinancialManagementSystem\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(46): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2022-12-03 07:18:14 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(250): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(623): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\SigninController.php(21): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\SigninController->loginAuth()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SigninController))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-12-03 09:11:25 --> Undefined variable $details
in APPPATH\Views\page\transaction_event.php on line 82.
 1 APPPATH\Views\page\transaction_event.php(82): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $details', 'APPPATH\\Views\\page\\transaction_event.php', 82)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\page\\transaction_event.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1134): CodeIgniter\View\View->render('page/transaction_event', [], true)
 5 APPPATH\Controllers\TransactionController.php(9): view('page/transaction_event')
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\TransactionController->AllBookings()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TransactionController))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-12-03 09:12:29 --> Invalid file: page/AllBooings.php
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('page/AllBooings.php')
 2 SYSTEMPATH\Common.php(1134): CodeIgniter\View\View->render('page/AllBooings', [], true)
 3 APPPATH\Controllers\TransactionController.php(9): view('page/AllBooings')
 4 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\TransactionController->AllBookings()
 5 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TransactionController))
 6 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-12-03 09:23:44 --> Invalid file: include/end.php
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('include/end.php')
 2 SYSTEMPATH\View\View.php(431): CodeIgniter\View\View->render('include/end', null, true)
 3 APPPATH\Views\page\AllBookings.php(103): CodeIgniter\View\View->include('include/end')
 4 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\page\\AllBookings.php')
 5 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 6 SYSTEMPATH\Common.php(1134): CodeIgniter\View\View->render('page/AllBookings', [], true)
 7 APPPATH\Controllers\TransactionController.php(9): view('page/AllBookings')
 8 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\TransactionController->AllBookings()
 9 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TransactionController))
10 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-12-03 09:54:39 --> Call to undefined method App\Controllers\ProfileController::include()
in APPPATH\Controllers\ProfileController.php on line 12.
 1 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\ProfileController->index()
 2 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProfileController))
 3 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-12-03 10:13:15 --> Undefined property: App\Controllers\ProfileController::$load
in APPPATH\Controllers\ProfileController.php on line 13.
 1 APPPATH\Controllers\ProfileController.php(13): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: App\\Controllers\\ProfileController::$load', 'APPPATH\\Controllers\\ProfileController.php', 13)
 2 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\ProfileController->index()
 3 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProfileController))
 4 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-12-03 10:13:59 --> Call to undefined method App\Controllers\ProfileController::view()
in APPPATH\Controllers\ProfileController.php on line 13.
 1 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\ProfileController->index()
 2 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProfileController))
 3 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
ERROR - 2022-12-03 10:20:55 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('localhost', 'root', '', 'financialdb', 3306, '', 0)
#1 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\Model.php(250): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\BaseModel.php(623): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\FinancialManagementSystem\app\Controllers\SigninController.php(21): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\SigninController->loginAuth()
#8 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SigninController))
#9 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\FinancialManagementSystem\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(46): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\Model.php(250): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\BaseModel.php(623): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\FinancialManagementSystem\app\Controllers\SigninController.php(21): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\SigninController->loginAuth()
#7 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SigninController))
#8 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\FinancialManagementSystem\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(46): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2022-12-03 10:20:55 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(250): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(623): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\SigninController.php(21): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\SigninController->loginAuth()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SigninController))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-12-03 10:36:56 --> Invalid file: dashboard.php
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('dashboard.php')
 2 SYSTEMPATH\Common.php(1134): CodeIgniter\View\View->render('dashboard', [], true)
 3 APPPATH\Controllers\ProfileController.php(14): view('dashboard')
 4 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\ProfileController->index()
 5 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProfileController))
 6 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
ERROR - 2022-12-03 17:34:23 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('localhost', 'root', '', 'financialdb', 3306, '', 0)
#1 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\Model.php(250): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\BaseModel.php(623): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\FinancialManagementSystem\app\Controllers\SigninController.php(21): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\SigninController->loginAuth()
#8 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SigninController))
#9 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\FinancialManagementSystem\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(46): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\Model.php(250): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\BaseModel.php(623): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\FinancialManagementSystem\app\Controllers\SigninController.php(21): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\SigninController->loginAuth()
#7 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SigninController))
#8 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\FinancialManagementSystem\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\FinancialManagementSystem\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(46): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2022-12-03 17:34:23 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(250): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(623): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\SigninController.php(21): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\SigninController->loginAuth()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SigninController))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
